### ISSUE TITLE
Debug on this plugin

### ISSUE THREAD

**Issue by dingye18** [pandegroup/openmm-nn#16]
Hello, I'm developing an openmm plugin based on this plugin. However, I don't know how to debug on this plugin cause everything goes well when I add this force to system. I don't know if the simulation used my plugin or not. Is there any useful tricks can be used for the debugging during simulation?

**Comment by peastman** [comment]
If you put the force into its own force group, you can query it separately.  When you first create it (and *before* creating the Context),

```python
nnforce.setForceGroup(1)
```

And then later you can query the energy of just that force group with

```python
state = context.getState(getEnergy=True, groups={1})
```

That way you can easily see what its energy is.  Or if you specify `getForces=True` you can query the forces it's applying.

**Comment by dingye18** [comment]
Thanks for your reply. But I have two questions here. First is that I did used `nnforce.setForceGroup(1)` after I create it. However, the output force with 
`state = context.getState(getEnergy=True, groups={1})`
is not reasonable with my plugin's output force.
Besides, when I iterate through `system.getForce(force_index)`, I found my nnforce did exist in system force groups. 
And then, I used this nnforce index instead of the 1 in 
`state = context.getState(getEnergy=True, groups={nnforce_index})`
The output force is zero at all atoms. It seems like the context did not calculate the nnforce in force groups, which is very confusing for me.......
